new_careers_api/config/prod.exs

```diff
@@ -15,7 +15,7 @@ config :new_careers_api, NewCareersApiWeb.Endpoint,
   version: Application.spec(:phoenix_distillery, :vsn),
   http: [:inet6, port: {:system, "PORT"}],
   load_from_system_env: true,
-  url: [host: "newcareers.tsengjonathan.com", port: 443],
+  url: [host: "newcareers.cs4550.maineisland.co", port: 443],
```

Comment:
Are we deploying on your VPS now? Just making sure

Reply:
Yeah it’s running fine right now so I think it will work